matcher/service/config/celery_config.py

```diff
@@ -0,0 +1,22 @@
+from kombu import Exchange, Queue
+from kombu.serialization import registry
+
+task_default_queue = 'default'
+task_queues = (
+    Queue('matching', routing_key='matcher.service.celery_tasks.run_matching'),
+    Queue('deployment', routing_key='matcher.service.celery_tasks.run_deployment')
+)
+# CELERY_IMPORTS = ('tasks')
+task_ignore_result = False
+# broker_url = 'redis://localhost:6379/0'
+broker_url = 'amqp://openreview:openreview@localhost:5672/localhost'
+result_backend = 'redis://localhost:6379/0'
```

Comment:
is redis a dependency?

Reply:
The celery documentation page says that you can use either RabbitMQ or Redis as broker, why did you choose RabbitMQ? which are the differences between them?

https://docs.celeryproject.org/en/stable/getting-started/first-steps-with-celery.html#redis

Reply:
I changed the broker to Redis, but while running tests I had to add 1 sec break between sending post requests. This seems to be just a celery worker pytest fixture issue as the test pass without the 1 sec break, if I start an actual celery worker. 

Nonetheless, I'd still suggest using RabbitMQ just because it's more reliable.